x-pack/plugins/cases/public/components/user_profiles/translations.ts

```diff
@@ -65,3 +65,10 @@ export const INVALID_ASSIGNEES = i18n.translate('xpack.cases.create.invalidAssig
     maxAssignees: MAX_ASSIGNEES_PER_CASE,
   },
 });
+
+export const MAX_SELECTED_ASSIGNEES = (limit: number) =>
+  i18n.translate('xpack.cases.userProfile.maxSelectedAssignees', {
+    defaultMessage:
+      "You've selected the maximum number of {count, plural, one {# assignee} other {# assignees}}",
```

Comment:
Is it really possible to have a maximum of 1? If yes, I think this wording is clearer:

```suggestion
      "You've selected the maximum number of assignees ({count}).",
```

Reply:
Thanks for the feedback. Right now it's configured to 10, I don't foresee us switching it to 1 but this function's implementation would allow us to. If we set it lower in the future we can change the wording then 👍